composer.json

```diff
@@ -30,5 +30,10 @@
       "Peridot\\": "src/"
     }
   },
+  "autoload-dev": {
```

Comment:
Would this be ignored by anyone requiring Peridot via composer? My concern here is that our build will pass as it runs `bin/peridot` but anyone requiring via composer would not have this dependency loaded. @ezzatron can you confirm?